app/assets/javascripts/discourse/app/widgets/quick-access-panel.js

```diff
@@ -40,7 +33,7 @@ export default createWidget("quick-access-panel", {
   },
 
   hasMore() {
-    return this.getItems().length >= this.estimateItemLimit();
+    return true;
```

Comment:
It looks like with this refactor you don't need `hasMore()` or `estimateItemLimit()` at all.